tutorials/qdrant_langchain_instrumentation_search_and_retrieval_tutorial.ipynb

```diff
@@ -0,0 +1,478 @@
+{
```

Comment:
<p>Did you mean to delete the diagram? If so, I might rephrase "shown below".</p><p> </p><p>Second list is confusing, I would delete the first bullet.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/phoenix/pull/2634/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>